src/libcore/option.rs

```diff
@@ -188,10 +188,10 @@ impl<T> Option<T> {
     /// # Example
     ///
     /// ```
-    /// let x: Option<uint> = Some(2);
+    /// let x: Option<usize> = Some(2);
     /// assert_eq!(x.is_some(), true);
     ///
-    /// let x: Option<uint> = None;
+    /// let x: Option<usize> = None;
```

Comment:
Some of these can actually be inferred if we want i32, although I dunno if that would "mess up" the quality of the docs.